etc/rc1

```diff
@@ -86,7 +86,7 @@ fi
 
 modload all job-ingest
 modload 0 job-exec
-modload 0 heartbeat
+modload all heartbeat
```

Comment:
The statement in the commit message "the new heartbeat timer requires it to be loaded on all ranks" confused me. Is it actually a future commit that will require it be to loaded on all ranks, or should the message read something like "is supported on all ranks"?

Reply:
I'll reword.  I meant the future commit.